SDK/AppCenterAuth/Microsoft.AppCenter.Auth.iOS.Bindings/ApiDefinition.cs

```diff
@@ -35,12 +35,12 @@ interface MSAuth
         [Export("signOut")]
         void SignOut();
 
-        // + (void)openURL:(NSURL *)url;
+        // + (BOOL)openURL:(NSURL *)url options:(NSDictionary<UIApplicationOpenURLOptionsKey, id> *)options;
         [Static]
-        [Export("openURL:")]
-        void OpenUrl(NSUrl url);
+        [Export("openURL:options:")]
+        void OpenUrl(NSUrl url, NSDictionary options);
```

Comment:
Why not generic?

Reply:
https://docs.microsoft.com/en-us/dotnet/api/uikit.uiapplicationdelegate.openurl?view=xamarin-ios-sdk-12